releasenotes/notes/fix-inverse-cancellation-self-inverse-e09a5553331e1b0b.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+fixes:
+  - |
+    :class:`InverseCancellation` transpilation pass will now correctly handle self-inverse
+    parameterized gates. `#11815 <https://github.com/Qiskit/qiskit/issues/11815>`__ is now fixed.
```

Comment:
```suggestion
    Fixed an issue in the :class:`.InverseCancellation` transpiler pass where it would incorrectly handle
    some self-inverse parameterized gates.
    Fixed  `#11815 <https://github.com/Qiskit/qiskit/issues/11815>`__
```